src/Features/Core/Portable/Completion/CompletionChange.cs

```diff
@@ -41,9 +41,6 @@ private CompletionChange(ImmutableArray<TextChange> textChanges, int? newPositio
         private CompletionChange(TextChange textChange, int? newPosition, bool includesCommitCharacter)
         {
             TextChange = textChange;
-#pragma warning disable CS0618 // Type or member is obsolete
-            TextChanges = ImmutableArray.Create(textChange);
```

Comment:
Our feeling is that this API simply isn't one htat people will really be making use of.  It's too new, and it's still too difficult to be a roslyn service language *provider* (as opposed to consumer).